jib-core/src/main/java/com/google/cloud/tools/jib/image/json/JsonToImageTranslator.java

```diff
@@ -149,13 +161,8 @@
       imageBuilder.setOs(containerConfigurationTemplate.getOs());
     }
 
-    if (containerConfigurationTemplate.getContainerEntrypoint() != null) {
-      imageBuilder.setEntrypoint(containerConfigurationTemplate.getContainerEntrypoint());
-    }
-
-    if (containerConfigurationTemplate.getContainerCmd() != null) {
-      imageBuilder.setProgramArguments(containerConfigurationTemplate.getContainerCmd());
-    }
+    imageBuilder.setEntrypoint(containerConfigurationTemplate.getContainerEntrypoint());
+    imageBuilder.setProgramArguments(containerConfigurationTemplate.getContainerCmd());
```

Comment:
So previously we would only overwrite entrypoint/args if set in the container configuration.  The code further above continues doing this for `created`, `architecture` and `os`, and further below.  Is there a reason to deviate from the previous pattern?

Reply:
At some point, looks like we changed `setEntrypoint()`, `setProgramArguments()`, etc. to accept `@Nullable`. As a builder, `imageBuilder` starts with null values, and passing null to them is like no-op hence OK. It also felt a bit weird to do the a check when the method can accept null. Sure, if `setEntrypoint()` was called on `imageBuilder` with a concrete value before and if we pass null then the value will be gone, but calling `setEntrypoint()` twice with different values cannot happen here.